ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-ref.yaml

```diff
@@ -18,9 +18,13 @@ ref:
     documentation: |-
       Default is true, which means the workload with delete and recreate a set number of namespaces
   - name: EXTRA_FLAGS
-    default: "--churn-duration=20m"
```

Comment:
If you remove this default you do have to add to every consuming step/job. I believe there are some outside of openshift-qe dir, so I would not touch the default here personally.

Reply:
Added it back